gradle.properties

```diff
@@ -6,3 +6,5 @@ org.gradle.parallel=true
 # Apparently, IntelliJ-specific :buildSearchableOptions task (a dependency of :buildPlugin) relies on jars rather than class folders when building the plugin.
 # Without this setting, we end up with `NoClassDefFound` error for classes from Gradle modules that `frontend` depends on (e.g. `backendApi`).
 systemProp.org.gradle.java.compile-classpath-packaging=true
+
+org.gradle.jvmargs=-Xmx2048m
```

Comment:
Hmmm does it apply for each of the workers separately, or just for the main orchestrating process?...

Reply:
In the former case it might hard to fit into our current CI memory limits :/

Reply:
Also, what actually prompted you to make this change? Did you get an OOM locally?

Reply:
Whoops, yeah, it looks that Gradle can't spawn the worker daemons on CI, which would indicate that this option applies to each Gradle process (including workers) and not just the main orchestrator :/

Reply:
Yeah, it started with OOM locally. I was just playing with this option. Hoverwer the case was very unusual (I tried to build the project from the past, about version `0.0.1` :rofl: ) Change reverted